packages/next/src/lib/turbopack-warning.ts

```diff
@@ -34,6 +35,12 @@ const supportedTurbopackNextConfigOptions = [
   'distDir',
   'skipMiddlewareUrlNormalize',
   'skipTrailingSlashRedirect',
+  'experimental.manualClientBasePath',
```

Comment:
https://github.com/vercel/next.js/blob/5521d0ed828f54324f79b3e2d1093b6d4f51c2c2/packages/next-swc/crates/next-core/src/next_config.rs#L429
@timneutkens swc_plugins is supported but missing in this list.

Reply:
Thanks! Fixed in #56201 👍